english/marshmallow.md

```diff
@@ -606,4 +606,6 @@ Just then you recall the news story of what is causing the
 
 You escape into a [recording studio](more_cowbell/more_cowbell.md)
 
-And then you stumble upon something [completely different](http://www.atlasobscura.com/)
\ No newline at end of file
+And then you stumble upon something [completely different](http://www.atlasobscura.com/)
+
+A snarling ferret king who fixes you with a [steely glare](../ferrets/Ferret.md)
```

Comment:
The `../` at the beginning is unnecessary, since the `ferrets` directory is directly inside of the `english` directory.